### PR TITLE
Add bulk_published to aaib-report example

### PR DESCRIPTION
**Pull request by binaryberry**
This is following a conversation on specialist-frontend [bugfix PR](https://github.com/alphagov/specialist-frontend/pull/102#discussion_r38933024).

cc @tommyp @boffbowsh

### CODE REVIEW

File: formats/specialist_document/frontend/examples/aaib-reports.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
 			],
 			"report_type" : "correspondence-investigation",
 			"date_of_occurrence" : "2015-08-08",
+			"bulk_published" : false,
```
Comment (tommyp) [inline formats/specialist_document/frontend/examples/aaib-reports.json]: This needs added to the all the examples as any format can be `bulk_published`?

Comment (jamiecobbett) [inline formats/specialist_document/frontend/examples/aaib-reports.json]: If it isn't a required field, then it doesn't need to be in every example.

Comment (tommyp) [inline formats/specialist_document/frontend/examples/aaib-reports.json]: Ah yeah, good call. But it does need to be in the `metadata` section for all the formats.

Comment (boffbowsh) [inline formats/specialist_document/frontend/examples/aaib-reports.json]: Is there an easy way to add it to all formats? Some sort of hash merging thing?

Comment (tommyp) [inline formats/specialist_document/frontend/examples/aaib-reports.json]: It needs added to each of the metadata definitions [here](https://github.com/alphagov/govuk-content-schemas/blob/bulkpublishedfix/dist/formats/specialist_document/frontend/schema.json#L136-L163).

Comment (binaryberry) [inline formats/specialist_document/frontend/examples/aaib-reports.json]: Thanks @tommyp , added now!

File: formats/specialist_document/frontend/examples/cma-cases.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
...
 		"metadata": {
+		"bulk_published" : false,
```
Comment (tommyp) [inline formats/specialist_document/frontend/examples/cma-cases.json]: Indentation is off here.

Comment (binaryberry) [inline formats/specialist_document/frontend/examples/cma-cases.json]: True! Amended now :-)

**Comment by tommyp** [comment]
Nice work @binaryberry! 👍

**Comment by binaryberry** [comment]
Thanks for all your help @tommyp  !

**Pull request merged**